autogen.sh

```diff
@@ -48,4 +48,11 @@ gtkdocize || exit $?
 autoreconf -fi || exit $?
 
 cd "$olddir"
-test -n "$NOCONFIGURE" || "$srcdir/configure" "$@"
+
+# If no arguments passed, will use default arguments for eos-metrics.
+if [ $# -eq 0 ]
+    then
+        test -n "$NOCONFIGURE" || "$srcdir/configure" "--enable-strict-flags" "--enable-gtk-doc" "--enable-gir-doc"
```

Comment:
I would prefer to change this in `configure.ac` by making "yes" the default for each of these options instead of "no". People won't think to look in this file because it's not customary to add arguments here; let's keep them in one place.